SU-SWS/stanford_layouts
# EARTH-159 Added title region to render h1 tag

# READY FOR REVIEW

# Summary
- Created a "title" region in ibeam to use for page titles.

# Needed By (Date)
- End of sprint

# Steps to Test

1. Checkout branch and same branch in [SE3_BLT](https://github.com/stanford-earth/SE3_BLT/pull/43) & [stanford_news](https://github.com/SU-SWS/stanford_news/pull/54)
2. cache clear then Config Import, features import all
3. create a news content
4. populate the hero banner or the video
5. verify it looks correct and the page title displays if video used or hero banner used with nothing in the "super heading" field (with variant set to "page title").

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_news/pull/54
- https://github.com/stanford-earth/SE3_BLT/pull/43

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
